gnd/src/main/java/com/google/android/gnd/persistence/local/room/converter/ResponseJsonConverter.java

```diff
@@ -67,13 +66,15 @@ static Object toJsonObject(Response response) {
   }
 
   public static String dateToIsoString(Date date) {
-    return ISO_INSTANT_FORMAT.format(date);
+    DateFormat dateFormat = new SimpleDateFormat(ISO_INSTANT_FORMAT, Locale.getDefault());
```

Comment:
This can be defined as a constant as well.

Reply:
updated https://github.com/google/ground-android/pull/846/commits/a950d1b98f2dfde1ae77921c5ffd9d9d2a3b732b